config/values/widget.yml

```diff
@@ -0,0 +1,3 @@
+widget:
+  subject:
+    art-and-design-bursary: £10,000
```

Comment:
So in our prod implementation, this will be:
`art__and__design__bursary`

and that will also be reflected in the YML